raven-log4j/src/main/java/com/getsentry/raven/log4j/SentryAppender.java

```diff
@@ -110,6 +107,54 @@ public SentryAppender(Raven raven) {
     }
 
     /**
+     * Do some appender initialization *after* instance construction, so that we don't
+     * log in the constructor (which can cause annoying messages) and so that system
+     * properties and environment variables override hardcoded appender configuration.
+     */
+    @SuppressWarnings("checkstyle:hiddenfield")
+    private void lazyInit() {
```

Comment:
One of the constructors takes `Raven` as a parameter, so `raven` being not-null isn't enough of a check for this.